File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/ManualDexterityTestsFormViewModel.ts
//$FBC749FB
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { ManualDexterityTestsForm } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class ManualDexterityTestsFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.ManualDexterityTestsFormViewModel, Core';
    @Type(() => ManualDexterityTestsForm)
    public _ManualDexterityTestsForm: ManualDexterityTestsForm = new ManualDexterityTestsForm();
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/ManualDexterityTestsFormViewModel.cs
//$4F8E3A90
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class ManualDexterityTestsFormServiceController
    {
    }
}

namespace Core.Models
{
    public partial class ManualDexterityTestsFormViewModel: BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var manualDexterityTestsFormService = new ManualDexterityTestsFormServiceController();
            manualDexterityTestsFormService.ManualDexterityTestsFormInternal(this, objectContext);
            return (ManualDexterityTestsForm)objectContext.GetObject(this._ManualDexterityTestsForm.ObjectID, "ManualDexterityTestsForm");
        }
    }
}
